uds/uds_config_tool/odx/pos_response.py

```diff
@@ -0,0 +1,80 @@
+from typing import Dict, List
+
+from uds.uds_config_tool import DecodeFunctions
+from uds.uds_config_tool.odx.param import Param
+
+
+class PosResponse():
+    """encapsulates PARAMs and DID + SID information for parsing and decoding a positive uds response
+    """
+
+    def __init__(self, params: List[Param], didLength: int, DID: int, sidLength: int, SID: int) -> None:
+        """initialize attributes
+
+        :param params: List of Params that are defined in the POS-RESPONSE for a DID
+        :param didLength: byte length of the PosResponse's DID
+        :param DID: the DID value
+        :param sidLength: byte length of the PosResponse's SID
+        :SID: the SID value
+        """
+        self.params = params
+        self.didLength = didLength
+        self.DID = DID
```

Comment:
```suggestion
        self.did = did
```
These aren't constants so shouldn't be uppercase

Reply:
fixed for DID and SID